src/core/style/themes.scss

```diff
@@ -39,4 +39,10 @@ html, body {
   &#{$checkedSelector} ._md-icon:after {
     border-color: '{{primary-contrast-0.87}}';
   }
+
+  & .md-indeterminate[disabled] {
```

Comment:
We created a mixin for the checkbox's css so it could be used in other components. This is where all the css goes now that might be used by multiple components.